app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_when_age_less_14() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
```

Comment:
Instead of writing a few tests, you can use @pytest.mark.parametrize to transfer parameters into a function. 
https://docs.pytest.org/en/7.2.x/how-to/parametrize.html
And also maybe check how the function will work with unusual parameters( for ex negative numbers or zero)